### PR TITLE
[ken-schedule-v2] Make aumni logo first and larger; add logos to schedule page

### PR DESCRIPTION
**Pull request by kensnyder**
Longer term, I want to add sort order and tier (premier, gold, silver, bronze). For now, this just uses CSS to make the aumni logo larger.

image

### CODE REVIEW

File: gatsby/src/components/ConferenceSponsors.js [hunk 1]
```diff
@@ -14,12 +14,16 @@ const Wrapper = styled.div`
   .wrapper {
     border-top: 2px dotted #ccc;
     display: flex;
-    flex-direction: column;
+    flex-direction: column-reverse;
```
Comment (kensnyder) [inline gatsby/src/components/ConferenceSponsors.js:17]: makes aumni logo first.

File: gatsby/src/components/ConferenceSponsors.js [hunk 2]
```diff
@@ -14,12 +14,16 @@ const Wrapper = styled.div`
...
     align-items: center;
   }
   .spacer {
     margin-top: 20px;
   }
+  a[href*='aumni.fund'] {
+    transform: scale(1.5);
+    display: block;
+  }
```
Comment (kensnyder) [inline gatsby/src/components/ConferenceSponsors.js:26]: makes aumni logo larger

File: gatsby/src/pages/schedule.js [hunk 3]
```diff
@@ -3,14 +3,31 @@ import { graphql } from 'gatsby';
...
+    justify-content: space-between;
+  }
+  h1 {
+    padding: 0 40px 0 0;
+    margin: 22px 0 0 0;
+  }
+  .sponsors-column {
+    transform: scale(0.75);
+    transform-origin: right top;
+  }
```
Comment (kensnyder) [inline gatsby/src/pages/schedule.js:30]: CSS for sponsors on schedule: show in a right column (except on mobile show underneath). Shrink content a bit so logos are not so huge.

File: gatsby/src/pages/schedule.js [hunk 4]
```diff
@@ -100,8 +117,12 @@ export default function Schedule({ data }) {
...
       <SEO seo={seo} />
       <PageStyles className="center-content">
-        <div className="padding" />
-        <h1>Conference Schedule: Friday, October 8, 2021</h1>
+        <div className="headings">
+          <h1>Conference Schedule: Friday, October 8, 2021</h1>
+          <div className="sponsors-column">
+            <Sponsors />
+          </div>
+        </div>
```
Comment (kensnyder) [inline gatsby/src/pages/schedule.js:125]: Adding sponsor component to display list of sponsors.

**Comment by AlexanderPuhl** [comment]
Everything looks good, no errors when I ran it locally.

**Pull request merged**